### PR TITLE
ChangeLog_creation_1.1.0

### PR DESCRIPTION
**Pull request by jjeroch**
ChangeLog for 1.1.0 created

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,28 +3,37 @@
...
-   * Service Release Steps
-* Application Request
-   * BugFix - Approve/Decline/Retrigger/Cancel Process actions should clear button and update the latest status in both overlay and application list
-* Bugfix - UI fixes in Adminboard
-* Connector
-   * BugFix - Register daps via the auth icon in the connector registration, allows all type of files
-* Notification
-   * BugFix - Wrong url link
 * Static Templates
    * Show image on modal for a wider view
```
Comment (jjeroch) [inline CHANGELOG.md:7]: @manojava-gk  please check the release log regarding "Static Template" - I unfortunately cant follow what the bullet point is supposed to say.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,28 +3,37 @@
...
+  * app admin board - app detail page enhanced by adding conformity document view and download
+* SharedUIComponents
+  * image gallery - added grid layout & parameter to display images in 16:9 ratio
+* Service Release Management
+  * implemented new top navigation section und sub-menu
+  * implemented service release process intro page
+  * implemented service release form (initial)
+
+### Technical Support
+* "allProps" replaced by intelligent image load handling
```
Comment (jjeroch) [inline CHANGELOG.md:30]: @oyo I am pretty sure I havnt named the "prop" change correctly, can you please rephrase the bullet point?

File: CHANGELOG.md (hunk 1 above)
Comment (manojava-gk) [reply CHANGELOG.md:7]: This about -> Option to zoom into pictures to get them bigger displayed on overlay

**Review by evegufy — APPROVED** [review]

**Comment by evegufy** [comment]
Will be merged https://github.com/catenax-ng/tx-portal-frontend/pull/258

**Pull request closed without merging**